src/session.ts

```diff
@@ -336,7 +344,11 @@ export class Session extends ServiceObject {
    *   }
    * });
    */
-  keepAlive(callback?): void | Promise<void> {
+  keepAlive(): Promise<[r.Response]>;
+  keepAlive(callback: NormalCallback<r.Response>): void;
+  keepAlive(
+    callback?: NormalCallback<r.Response>
```

Comment:
As per comment above `r.Response` should not be used
Please declare an interface for callback and response. Perhaps next to 
```ts
export type KeepAliveCallback = NormalCallback<
  spanner_client.spanner.v1.IResultSet
>;
export type KeepAliveResponse = [
  spanner_client.spanner.v1.IResultSet
];